### PR TITLE
Import ground markers

### PR DESCRIPTION
**Pull request by ConorLeckey**
No longer using ground marker tiles.

Added ability to convert ground markers into tileman tiles

### CODE REVIEW

File: src/main/java/com/tileman/TilemanModePlugin.java [hunk 1]
```diff
@@ -213,15 +225,70 @@ protected void shutDown() {
...
+            // If region already exists in Tileman World Regions Array:
+            if(tilemanModeRegions.contains(region)) {
+                // Create Empty ArrayList for Region;
+                // Get Tileman Region's tiles and add them to the region array list
+                ArrayList<TilemanModeTile> regionTiles = new ArrayList<>(getTiles(Integer.parseInt(region)));
+
+                // Create int for regionOriginalSize;
+                // Set regionOriginalSize to arraylists length
+                int regionOriginalSize = regionTiles.size();
+                int nonUniqueTiles = 0;
```
Comment (sram1337) [inline src/main/java/com/tileman/TilemanModePlugin.java:251]: Can delete this. Doesn't look like we use it anywhere

File: src/main/java/com/tileman/TilemanModePlugin.java [hunk 2]
```diff
@@ -213,15 +225,70 @@ protected void shutDown() {
...
+
+                // Loop through Ground Markers Points
+                for(TilemanModeTile groundMarkerTile: groundMarkerTiles) {
+                    // If Ground Marker point already exists in Tileman World Region: Break loop iteration
+                    if(regionTiles.contains(groundMarkerTile)){
+                        nonUniqueTiles++;
```
Comment (sram1337) [inline src/main/java/com/tileman/TilemanModePlugin.java:257]: here too

File: src/main/java/com/tileman/TilemanModePlugin.java [hunk 3]
```diff
@@ -213,15 +225,70 @@ protected void shutDown() {
...
+                    savePoints(Integer.parseInt(region), regionTiles);
+                }
+            } else {
+                // Save points for that region
+                savePoints(Integer.parseInt(region), groundMarkerTiles);
+            }
+        }
+    }
+
+    private List<String> removeRegionPrefixes(List<String> regions) {
```
Comment (sram1337) [inline src/main/java/com/tileman/TilemanModePlugin.java:275]: I swear we already had a method to do this...

**Comment by sram1337** [comment]
I'm not seeing the panel when I test locally :(

File: src/main/java/com/tileman/TilemanModePlugin.java (hunk 1 above)
Comment (ConorLeckey) [reply src/main/java/com/tileman/TilemanModePlugin.java:251]: Removing, was just using for debugging

File: src/main/java/com/tileman/TilemanModePlugin.java (hunk 3 above)
Comment (ConorLeckey) [reply src/main/java/com/tileman/TilemanModePlugin.java:275]: Not a method, but we had something similar happening in the updateTileCounter method, making it use this functionality instead (added another method to remove region prefix for individual region strings)

**Pull request merged**